### ISSUE TITLE
Importing the project in Eclipse removes important classpaths

### ISSUE THREAD

**Issue by simark** [TypeFox/tracing_codegen#2]
When importing the projects in Eclipse, some important classpaths get removed from the project's `.classpath` files.  I choose to import an "Existing Gradle Project" and point it to `.../tracing_codegen/io.typefox.traceexample.parent`.  The projects are successfully imported, but I can already see that Eclipse has removed some paths from the classpath:

```patch
diff --git a/io.typefox.traceexample.parent/io.typefox.traceexample.ide/.classpath b/io.typefox.traceexample.parent/io.typefox.traceexample.ide/.classpath
index 590bec1..43c5e2c 100644
--- a/io.typefox.traceexample.parent/io.typefox.traceexample.ide/.classpath
+++ b/io.typefox.traceexample.parent/io.typefox.traceexample.ide/.classpath
@@ -1,8 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
        <classpathentry kind="src" path="src/main/java"/>
-       <classpathentry kind="src" path="src/main/resources"/>
-       <classpathentry kind="src" path="src/main/xtext-gen"/>
        <classpathentry kind="src" path="src/main/xtend-gen"/>
        <classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8/"/>
        <classpathentry kind="con" path="org.eclipse.buildship.core.gradleclasspathcontainer"/>
diff --git a/io.typefox.traceexample.parent/io.typefox.traceexample/.classpath b/io.typefox.traceexample.parent/io.typefox.traceexample/.classpath
index 7d14c46..2bd854a 100644
--- a/io.typefox.traceexample.parent/io.typefox.traceexample/.classpath
+++ b/io.typefox.traceexample.parent/io.typefox.traceexample/.classpath
@@ -1,12 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
        <classpathentry kind="src" path="src/main/java"/>
-       <classpathentry kind="src" path="src/main/resources"/>
-       <classpathentry kind="src" path="src/main/xtext-gen"/>
        <classpathentry kind="src" path="src/main/xtend-gen"/>
        <classpathentry kind="src" path="src/test/java"/>
-       <classpathentry kind="src" path="src/test/resources"/>
-       <classpathentry kind="src" path="src/test/xtext-gen"/>
        <classpathentry kind="src" path="src/test/xtend-gen"/>
        <classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8/"/>
        <classpathentry kind="con" path="org.eclipse.buildship.core.gradleclasspathcontainer"/>
```

The end result is that the build artifacts that end up in the xtext-gen directories will be ignored.  It is then required to either revert those changes, or manually add the paths as source folders in the projects properties.

**Comment by kittaakos** [comment]
@simark Thanks for logging! Also, we want to get rid of this too: https://github.com/TypeFox/tracing_codegen/blob/master/io.typefox.traceexample.parent/io.typefox.traceexample/.classpath#L13

**Comment by kittaakos** [comment]
I'll look into this task.

**Comment by kittaakos** [comment]
Fixed via #3.

**Comment by simark** [comment]
I tried a fresh import and it works well, thanks!
